test/test.js

```diff
@@ -351,4 +361,32 @@ describe('request-progress', function () {
         expect(request.progressContext).to.be.an('object');
         expect(request.progressState).to.be(null);
     });
+
+    it('should ignore "data" event from the request', function (done) {
```

Comment:
Additionally, add a comment inside the block saying `// See: https://github.com/IndigoUnited/node-request-progress/issues/20`